alacritty/src/logging.rs

```diff
@@ -28,6 +28,9 @@ const ALACRITTY_LOG_ENV: &str = "ALACRITTY_LOG";
 /// Logging target for config error messages.
 pub const LOG_TARGET_CONFIG: &str = "alacritty_config_derive";
 
+/// Logging target for winit events.
+pub const LOG_TARGET_WINIT: &str = "winit";
```

Comment:
How frequently does winit log? I could see this easily going out of control.

Reply:
Could go for `alacritty_winit_event` just to be precise that this isn't actually logged from winit?

There's zero reason to be brief here, since it's always used through the constant anyway.